test/jdk/java/lang/constant/ClassDescTest.java

```diff
@@ -261,6 +265,17 @@ public void testBadClassDescs() {
             }
         }
 
+        List<String> badInternalNames = List.of("I;", "[]",
```

Comment:
is `[]` a good test for arrays? Wouldn't something more realistic like `[Ljava/lang/String` be better? I note that `[]` is also used in another test. Problem with testing `[]` is that if the checking logic has a bug, or a regression is introduced so that it only detects `]` instead of `[` the test would not detect that (but I guess that's a remote possibility).

Reply:
Thanks for pointing it out, I'll add a valid array descriptor into the list of badInternalNames.

Reply:
done, thanks